### DISCUSSION TITLE
[keyball39] 一部キーが反応しない

### DISCUSSION THREAD

**Discussion by gohan5858** [Yowkees/keyball#589] [Q&A]
当方、初めての自作キーボードで知識が乏しくお助け頂きたいです。
組み立てが終わり、ファームウェアを書き込み(片側づつPCに繋いでの)テストをしてみたのですが特定のキー(写真1枚目の灰色の部分)が反応しません。

![key_test](https://github.com/Yowkees/keyball/assets/88976739/2aa35982-d119-42fe-a509-06076d54d200)
![81063_0](https://github.com/Yowkees/keyball/assets/88976739/5e1e3fa5-ef39-4fc8-b48c-c705479c7bf1)
![81062_0](https://github.com/Yowkees/keyball/assets/88976739/811a800f-3a80-4c53-aade-ebbda720a0ff)

写真から問題がありそうなところがありましたら教えていただけると幸いです。
お手数おかけしますがよろしくお願いいたします。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

反応のないキーがバラバラとしていますので、キーソケット自体のはんだ付けが原因となっていると思われます。
結構根気よく作業する必要があるので、横から見て端子にしっかりはんだが馴染むようにはんだ付けを行ってください。

**Reply by gohan5858** [reply]
承知いたしました。
該当キーソケットのはんだ付けをやり直してみます。
ありがとうございます。

**Reply by gohan5858** [reply]
先ほど、該当キーソケットのはんだ付けをやり直したところ全てのキーが正常に反応するようになりました！
ありがとうございます！！

**Comment by apio1969** [comment]
こんにちは。
私も初めてKeyBallを組み立てたときはいくつもキーが反応しないことがありました。私の経験の中で、ランダムにキーが反応しなかった中にで半田付けの他によく起こったことは、キースイッチをはめ込む際に、爪を曲げてしまい、うまくソケットに刺さっていなかったことです。
反応しないキースイッチを一度取り外してみて、爪が曲がっていないかも確認されてはいかがでしょうか？
慎重にはめ込んだつもりでも、意外と爪が曲がってたりしました。

**Reply by gohan5858** [reply]
承知いたしました。
キースイッチのはんだ付けを修正する際に合わせて確認致します。
ありがとうございます。

**Comment by gohan5858** [comment]
無事解決いたしましたのでCloseします。
お助け頂いた[Ryo624](https://github.com/Ryo624)様、[apio1969](https://github.com/apio1969)様本当にありがとうございます！

[該当ディスカッション](https://github.com/Yowkees/keyball/discussions/589#discussioncomment-9501761)
